docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,83 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` |user |set certain contacts as favourites |access to their information easily
+
+|`* * *` |user |select multiple contacts to delete |clean up address book efficiently
+
+|`* * *` |user |save more than one contact number per contact in my address book |account for multiple phone numbers
+
+|`* * *` |organised user |group contacts manually or automatically |access the information I want easily
+
+|`* * *` |user |see my important events on a calendar view |overview of my schedule for the month
+
+|`* * *` |user |share my contact details with my friends |friends do not need to manually add it in
+
+|`* * *` |user |sort the information on the application |view the information in a ordered manner
+
+|`* * *` |user |add information to the application in a intuitive way |not frustrating
+
+|`* * *` |user |sync my account with other platforms |save the trouble of entering my personal details
+
+|`* * *` |user |access my data over multiple devices |do not have key in my data repeatedly
+
+|`* * *` |user |have my account logged in every time |do not have to remember one additional username and password
+
+|`* * *` |user |set reminders |do not forget what I need to do
+
+|`* * *` |user |export the list of people in my address book |store it on the computer
+
+|`* * *` |user |set events and send the event invites to any person in my contact list |calendars can sync the events
+
+|`* * *` |user |attach personal notes to people in my contact list |take notes about the person for my own reference
+
+|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else
+seeing them by accident
+
+|`* *` |user |send mass messages to a group of contacts |plan for meetings, conferences or gatherings
+
+|`* *` |user |see my groupings in a more visual display |see who are also in the group
+
+|`* *` |user |commands to be in the button form |don't have to memorise the command and it is easier to select
+
+|`* *` |user |have meta-data for my contact list |sort them according to different criteria
+
+|`* *` |user |the application should not take up too much space on my device |space for other applications
+
+|`* *` |user |the application should suggest certain fields for autocomplete |do not have to type them
+
+|`* *` |user |easily send emails to my contacts |more convenient to contact my friends
+
+|`* *` |user |update my location |contacts will know where is my current location
+
+|`* *` |user |update my status |contacts will know whether I’m busy
+
+|`* *` |user |view live stories of my friends |stay updated on what’s going on with my friends
+
+|`* *` |user |start a live chat with my friends |get an instant response from them
+
+|`* *` |user |add a profile picture |others will know how I look like
+
+|`* *` |user |change the colour scheme |make it personalized
+
+|`* *` |user |choose to require a password every time the application is opened |application stays secure
```

Comment:
choose not to cache the password after every login | ensure that the application is secure

Reply:
Done